cdc/processor/sinkmanager/redo_cache.go

```diff
@@ -56,6 +61,7 @@ func (r *redoEventCache) getAppender(tableID model.TableID) *eventAppender {
 // pop some events from the cache.
 func (r *redoEventCache) pop(
 	tableID model.TableID,
+	pushCount *int, // poped events come from how many `pushBatch` calls
```

Comment:
Can we make this a return value instead of an int pointer? It seems difficult to understand this API.

Reply:
Here it's expected. If callers doesn't care it, they can pass `nil` in.